build/site/specifications/2.1/specs/C2PA_Specification.html

```diff
@@ -1331,7 +1331,7 @@ <h4 id="_2_1_september_2024"><a class="anchor" href="#_2_1_september_2024"></a>5
 <div class="sect3">
 <h4 id="_2_0_january_2024"><a class="anchor" href="#_2_0_january_2024"></a>5.2.2. 2.0 - January 2024</h4>
 <div class="paragraph">
-<p>This version represents a significant departure from previous versions. It reduces the use of the term "actor", which no longer represents humans and organisations. In addition to validator-configured trust lists, it also introduces a new default trust list, the "C2PA Trust List", which is intended to cover certificates issued to hardware and software.  This philosophical change led to the following functional changes in the specification:</p>
+<p>This version represents a significant departure from previous versions. It reduces the use of the term "actor", which no longer represents humans and organizations. In addition to validator-configured trust lists, it also introduces a new default trust list, the "C2PA Trust List", which is intended to cover certificates issued to hardware and software.  This philosophical change led to the following functional changes in the specification:</p>
```

Comment:
We use Int'l English (which uses `s` instead of `z` on many words) for our standard.